packages/frontend-api/src/Resources/config/services.yml

```diff
@@ -17,3 +17,8 @@ services:
     Shopsys\FrontendApiBundle\Model\Error\ErrorHandlerListener:
         tags:
             - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: 20}
+
+    price.resolver:
```

Comment:
this should not be necessary

Reply:
It is use in `ProductDecorator.types.yml` aka (`resolve: "@=service('price.resolver').resolveByProduct(value)"`) Without this I will have to use full fully qualified class name. Should I do it?

Reply:
I would prefer FQN then

Reply:
Ok, I fixed it https://github.com/shopsys/shopsys/pull/1471/commits/f558c0d12cfefdc48483515aa2a79464a34823d7